src/app/app.html

```diff
@@ -1,8 +1,13 @@
 <ion-split-pane>
   <ion-menu [content]="content">
-    <ion-header aria-hidden="true">
+    <ion-header>
       <ion-toolbar color="primary">
-        <ion-title>PVTrAck</ion-title>
+        <ion-title aria-label="Peeveetrack">PVTrAck</ion-title>
+        <ion-buttons end>
+        <button ion-button menuClose aria-label="Close the menu" tabindex=2 *ngIf="this.platform.is('core') == false">
```

Comment:
Looks like [runningInBrowser](https://github.com/umts/pvta-multiplatform/blob/master/src/app/app.component.ts#L69) is all I need?